Controller/ExceptionController.php

```diff
@@ -126,7 +126,7 @@ protected function getAndCleanOutputBuffering()
      * @param FlattenException $exception
      * @param array            $exceptionMap
      *
-     * @return int|false
+     * @return mixed False, or the $exceptionMap corresponding value
```

Comment:
this is wrong. The ``$exceptionMap`` array is meant to contain only integers as values